src/modm/processing/timer/timestamp.hpp

```diff
@@ -30,9 +31,10 @@ namespace modm
  *
  * @author	Fabian Greif
  * @author	Niklas Hauser
+ * @author	Raphael Lehmann
  * @ingroup	modm_processing_timer
  */
-template< typename T >
+template< typename T, int scale = 1 >
```

Comment:
Perhaps we can switch to `std::chrono::duration`? 
https://en.cppreference.com/w/cpp/chrono/duration

Reply:
https://en.cppreference.com/w/cpp/chrono/time_point would be the correct substitute here